.devops/llama-cpp.srpm.spec

```diff
@@ -34,13 +37,14 @@ Models are not included in this package and must be downloaded separately.
 %setup -n llama.cpp-master
 
 %build
-make -j
+cmake -B build -DBUILD_SHARED_LIBS=0
+cmake --build build --config Release -j$(nproc)
 
 %install
 mkdir -p %{buildroot}%{_bindir}/
-cp -p llama-cli %{buildroot}%{_bindir}/llama-cli
+cd build/bin
+ls
```

Comment:
Is this intended?

Reply:
Yes it is. Since the build uses a snapshot of master it's really helpful to list the bin contents in build logs. People keep changing build outputs etc and that helps me troubleshoot automation. Valid?